modules/ROOT/pages/ip-blacklist.adoc

```diff
@@ -7,44 +7,48 @@ endif::[]
 [width="100%", cols="5,15"]
 |===
 >s| Policy Name | IP Blacklist
->s|Summary      | Blocks a list or range of IP addresses
+>s|Summary      | Blocks access from a range of IP addresses to an API endpoint
 >s|Category | Security
->s| Since Mule Version | 3.8.0
+>s|First Mule version available  | 3.8.0
 .1+>.^s| Returned Status Codes
 | 403 - IP is rejected
 |===
 
-The IP blacklist policy can control whether a list or range of IP addresses can access and interact with a configured API endpoint. You use one of the following options to blacklist addresses for your API:
+The IP Blacklist policy controls access to a configured API endpoint from a range of IP addresses. 
 
-* Option 1: Blacklist access based on the IP address of the request
-* Option 2: Blacklist access based on the origin IP address of the x-Forwarded-For header
+The IP Blacklist policy restricts access to a protected resource when a match is found between a source IP (specified when configuring the policy) and a list of individual IPs or range of IPs.
 
-== Option 1: Blacklist access based on the IP address of the request
+Based on your business need, you can specify a DataWeave 2.0 expression for Mule runtime engine 4 (Mule)) or MEL expression for Mule 3.
```

Comment:
The reason of why could be useful to do that is missing. The expression can be specified to define which would be the source IP to be used when the policy tries to determine if the IP belongs to the restricted IP list defined in the policy or not.

Reply:
Done! Thanks!